client/src/modules/home/index.tsx

```diff
@@ -5,19 +5,43 @@ import {
   GridItem,
   Button,
   useToast,
+  Flex,
+  Text,
+  Link,
 } from '@chakra-ui/react';
 import React, { useState } from 'react';
 
 import { Loading } from 'components/Loading';
 import { ChapterCard } from 'components/ChapterCard';
 import { EventCard } from 'components/EventCard';
 import { useHomeQuery } from 'generated/graphql';
+import { AuthContextType, useAuth } from 'modules/auth/store';
 
+type User = NonNullable<AuthContextType['user']>;
+
+const Welcome = ({ user }: { user: User }) => {
+  const helloUser = `Hello, ${user.name || 'annoymous'}`;
+  return (
+    <>
```

Comment:
```suggestion
```

Sorry about that

Reply:
No worries, it's easily done.

If there are a lot of changes, it's a pain to submit them through the GitHub UI. So, feel free to push a commit when that will save you time.